src/gov-action-contracts/util/OfficeHoursAction.sol

```diff
@@ -2,33 +2,76 @@
 pragma solidity 0.8.16;
 
 /// @notice Action that requires the current time to be within office hours. Should be included as an L1 action in proposal data.
-/// @dev    Time is kept in UTC
 contract OfficeHoursAction {
-    uint256 public immutable minUtcHour;
-    uint256 public immutable maxUtcHour;
+    /// @notice The minimum hour (inclusive) to execute the action on. 0 = midnight, 23 = 11pm
+    /// @dev We assume the range would not cross local midnight
+    uint256 public immutable minLocalHour;
+    /// @notice The maximum hour (exclusive) to execute the action on. 0 = midnight, 23 = 11pm
+    /// @dev We assume the range would not cross local midnight
+    uint256 public immutable maxLocalHour;
+
+    /// @notice The offset from UTC to local time. e.g. -5 for EST, -4 for EDT
+    int256 public immutable localHourOffset;
```

Comment:
All UTC doesn't work well with time wrapping midnight unless we use a 48 hours notation or some conditional depending on if start < end. 

Having the offset is also easier for most people to read imo.

Reply:
Yeh, I guess in most cases this will be fine, the only case it wont be is when we want to cross midnight. Then the caller will need to set some random timezone